### DISCUSSION TITLE
The function f2c::Transform::transformToUTM(field) crashes!

### DISCUSSION THREAD

**Discussion by AndrewChaschin** [Fields2Cover/Fields2Cover#115] [Q&A] [answered]
Hi!
When transforming coordinates EPSG:6326 -> EPSG:4326.
*poCT = 0x00 

![2024-04-02 191527](https://github.com/Fields2Cover/Fields2Cover/assets/24415915/315d421c-6a06-4c4c-be91-0f1dd7d179b3)

Build os Android 11 NDK armeabi-v7a. 
Most likely there is an error in the createSptRef function.
Has anyone else encountered a similar problem?
Thanks

**Comment by Gonzalo-Mier** [comment, marked as answer]
Hi @AndrewChaschin, welcome to the project. Please, make an issue (not a discussion) with a simpe example that shows the error :)

**Comment by AndrewChaschin** [comment]
Hi [Gonzalo-Mier](https://github.com/Gonzalo-Mier) 
This was due to the peculiarities of searching for the proj.db file in the Android OS.
Also, NDK Android does not work well with regular expressions.
This is not a program error, but a porting experience.
